### ISSUE TITLE
possible use with webrtc

### ISSUE THREAD

**Issue by cocodrino** [mafintosh/airpaste#1]
Hi mathias, I'm curious, would be possible use airpaste for make the handshake in webrtc and doesnt need a dedicate server, basically would share video and audio with other persons in my  network..thanks

**Comment by mafintosh** [comment]
yes! that should be totally doable assuming you just wanna connect with people on the local network
